### DISCUSSION TITLE
Is there a way to jump to the bottom of the log?

### DISCUSSION THREAD

**Discussion by ghost** [datalust/seq-tickets#1139] [Help] [answered]
I have a very large log -- even when filtered down to what I need. And I need to get to the bottom of the log so I can start "stepping" through it to follow the logic.

Is there a way to jump to the end (last line)? Scrolling with the scroll bar or page down key is taking forEVER.

**Comment by nblumhardt** [comment, marked as answer]
Hi! Again not exactly :-) but dropping out the histogram control from just below the filter bar can make it easier to move backwards and forwards in "pages":

![image](https://user-images.githubusercontent.com/342712/108450014-2d045000-72b0-11eb-8ebb-b1ee3637bc4f.png)

(Button on the left)

Hope this helps!
